client/src/components/AddToCollectionSearch.jsx

```diff
@@ -2,12 +2,52 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux'
 import { Modal, FormControl, InputGroup, ListGroup } from 'react-bootstrap';
+import BootstrapTable from 'react-bootstrap-table-next';
+import paginationFactory from 'react-bootstrap-table2-paginator';
+import ToolkitProvider, { Search } from 'react-bootstrap-table2-toolkit';
 import Button from '@material-ui/core/Button';
 import AddIcon from '@material-ui/icons/AddCircle';
 import SearchIcon from '@material-ui/icons/Search';
-// import DotLoader from 'react-spinners/DotLoader';
 
-import { getCollection, getGames, addGameToCollection, removeGameFromCollection } from '../actions/index'
+import { getCollection, getGames, addGameToCollection, removeGameFromCollection } from '../actions/index';
+
+// Column names for table of games
+const tableColumns = [{
+  dataField: 'id',
+  text: 'Game ID',
+  hidden: true,
+  searchable: false
+}, {
+  dataField: 'name',
+  text: '',
+  hidden: true
+}, {
+  dataField: 'gameInfo',
+  text: 'Search results:',
+  searchable: false,
+  headerStyle: { fontWeight: 'bold' }
+}, {
+  dataField: 'actionButton',
+  text: '',
```

Comment:
Is the text intentionally blank for these?

Reply:
Yeah, it's a required field, but I didn't want to show a heading for the action button column (and then the first column is hidden anyway)